docs/additional-functionality/rapids-shuffle.md

```diff
@@ -8,13 +8,50 @@ nav_order: 5
 
 The RAPIDS Shuffle Manager is an implementation of the `ShuffleManager` interface in Apache Spark
 that allows custom mechanisms to exchange shuffle data. We currently expose two modes of operation:
-UCX and Multi Threaded (experimental). 
+Multi Threaded and UCX. 
+
+In Spark, shuffle managers are configured via the `spark.shuffle.manager` configuration variable. 
+The following table shows the appropriate configuration to use for each Spark version supported
+in our plugin:
+
+| Spark Shim      | spark.shuffle.manager value                              |
+| --------------- | -------------------------------------------------------- |
+| 3.1.1           | com.nvidia.spark.rapids.spark311.RapidsShuffleManager    |
+| 3.1.2           | com.nvidia.spark.rapids.spark312.RapidsShuffleManager    |
+| 3.1.3           | com.nvidia.spark.rapids.spark313.RapidsShuffleManager    |
+| 3.2.0           | com.nvidia.spark.rapids.spark320.RapidsShuffleManager    |
+| 3.2.1           | com.nvidia.spark.rapids.spark321.RapidsShuffleManager    |
+| 3.2.1 CDH       | com.nvidia.spark.rapids.spark321cdh.RapidsShuffleManager |
+| 3.2.2           | com.nvidia.spark.rapids.spark322.RapidsShuffleManager    |
+| 3.2.3           | com.nvidia.spark.rapids.spark323.RapidsShuffleManager    |
+| 3.3.0           | com.nvidia.spark.rapids.spark330.RapidsShuffleManager    |
+| 3.3.1           | com.nvidia.spark.rapids.spark331.RapidsShuffleManager    |
+| Databricks 9.1  | com.nvidia.spark.rapids.spark312db.RapidsShuffleManager  |
+| Databricks 10.4 | com.nvidia.spark.rapids.spark321db.RapidsShuffleManager  |
+
+## Multi-Threaded Mode
+
+This mode is similar to the built-in Spark shuffle, but it attempts to use more CPU threads
+for compute-intensive tasks, such as compression and decompression. 
+
+Minimum configuration:
+
+```shell
+--conf spark.shuffle.manager=com.nvidia.spark.rapids.[shim package].RapidsShuffleManager \
+--conf spark.driver.extraClassPath=${SPARK_RAPIDS_PLUGIN_JAR} \
+--conf spark.executor.extraClassPath=${SPARK_RAPIDS_PLUGIN_JAR} \
+--conf spark.rapids.shuffle.mode=MULTITHREADED
```

Comment:
we don't need this if its default now right?

Reply:
whoops right :)